core/src/main/scala/kafka/consumer/TopicCount.scala

```diff
@@ -120,6 +120,10 @@ private[kafka] class StaticTopicCount(val consumerIdString: String,
     }
   }
 
+  override def hashCode(): Int = {
```

Comment:
Good point.  I don't think anywhere is actually using this equals method... let me try re-running the tests with an equals() method that throws an exception.